src/components/CardListBox/CardFolderList/CardFolderList.tsx

```diff
@@ -0,0 +1,35 @@
+import Folder from "./Folder/Folder";
+import styles from "./CardFolderList.module.scss";
+import { ObjectFolder } from "@/constants/index.types";
+
+interface CardFolderListProps {
+  folders: ObjectFolder[];
+  onFolderClick: (folder: ObjectFolder) => void;
+  onTotalButtonClick: () => void;
+}
+
+export default function CardFolderList({
+  folders,
+  onFolderClick,
+  onTotalButtonClick,
+}: CardFolderListProps) {
+  return (
+    <section className={styles.CardFolderList}>
+      <button
+        className={styles.totalBtn}
+        onClick={onTotalButtonClick}
+        type="button"
+      >
+        전체
+      </button>
+      {folders &&
+        folders.map((folder) => (
```

Comment:
요기서는 folder가 무조건 들어오는 값이기 때문에 이렇게만 표현해주셔도 될 것 같아요~
```suggestion
      {folders.map((folder) => (
```